TwitchChannelPointsMiner/classes/AnalyticsServer.py

```diff
@@ -20,17 +20,21 @@ def streamers_available():
     ]
 
 
-def read_json(streamer):
+def read_json(streamer, baseurl):
     path = Settings.analytics_path
     streamer = streamer if streamer.endswith(".json") else f"{streamer}.json"
+    streamer_without_baseurl = streamer.replace(baseurl, "")
```

Comment:
Also just out of curiosity, what does `baseurl` contains? Something like `baseurl/json/streamer.json`? 

Because I don't really get why this replacement is needed.
To me it should be something like `streamer.json` as before there was no replacements being done and url was `/json/streamer.json`.

Reply:
It contains that, but if a redirect is set it could be removed. The only thing that would need to be changed is the index file, to add the base url there